openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -1855,7 +1844,7 @@ def __init__(
                     f"0.3 SMIRNOFF spec requires each parameter section to have its own version."
                 )
 
-        # List of ParameterType objects (also behaves like an OrderedDict where keys are SMARTS).
+        # list of ParameterType objects (also behaves like an Ordereddict where keys are SMARTS).
```

Comment:
```suggestion
        # list of ParameterType objects (also behaves like an OrderedDict where keys are SMARTS).
```

Reply:
I'd really like to remove `OrderedDict` from our cognitive space, I think I'll try a different change here